resources/transactions.ts

```diff
@@ -69,6 +69,11 @@ export class Transactions extends BaseResource {
     public async update(request: PatchTransactionRequest): Promise<UnitResponse<Transaction>> {
         return await this.httpPatch<UnitResponse<Transaction>>(`/accounts/${request.accountId}/transactions/${request.transactionId}`,{ data: request.data })
     }
+
+    public async updateBookOrChargeback(request: PatchBookOrChargebackRequest): Promise<UnitResponse<Transaction>> {
```

Comment:
let's rename the function to
updateWithRelationships

```suggestion
    public async updateWithRelationships(request: PatchTransactionWithRelationshipsRequest): Promise<UnitResponse<Transaction>> {
```